### ISSUE TITLE
UX idea - "skeleton" loading animaiton

### ISSUE THREAD

**Issue by shootermv** [freeCodeCamp/chapter#1330]
### Describe the bug:
currently when data loaded we can see triangle animation at button right corner
can we replace animation by something like this:
![image](https://user-images.githubusercontent.com/1336862/182857988-4733fc44-693e-42a9-8c2e-eb7dadcfe585.png)

### Tell us about your browser and operating system:
- Browser(s) name and version: 
- Operating System:

**Comment by Sboonny** [comment]
I think we should worry about making chapter load faster, before considering add loaders

**Comment by shootermv** [comment]
> I think we should worry about making chapter load faster first, before considering add loaders

i only can see that lot of modern apps like youtube, linkedin, discord - do have skeleton animation

**Comment by Sboonny** [comment]
Yes, they are wonderful, and I am in support of them. But we should seek to cure it first before deciding to live with it. 

I mean stuff that increase the performance, like image lazy load, ordering the rendering of the components so the important stuff render first, and the unimportant don't get rendered until interaction deemed it necessary, I am sure there are more options to take.

So we should take these options before we add loaders.
